package.json

```diff
@@ -33,10 +36,16 @@
   },
   "homepage": "https://github.com/gagern/node-libtidy#readme",
   "devDependencies": {
+    "@types/chai": "^3.5.0",
+    "@types/mocha": "^2.2.40",
+    "@types/node": "^7.0.12",
```

Comment:
This one is a bit tricky, `@types/` packages does not cover some of corresponding JS packages. Will see if I can fix or remove some packages.